tpu-client-next/src/connection_worker.rs

```diff
@@ -18,11 +18,16 @@ use {
     },
     tokio::{
         sync::mpsc,
-        time::{sleep, Duration},
+        time::{sleep, timeout, Duration},
     },
     tokio_util::sync::CancellationToken,
 };
 
+/// The maximum connection handshake timeout for QUIC connections.
+/// This is set to 2 seconds, which was the earlier shorter connection idle timeout
+/// which was also used by QUINN to timemout connection handshake.
+pub(crate) const DEFAULT_MAX_CONNECTION_HANDSHAKE_TIMEOUT: Duration = Duration::from_secs(2);
```

Comment:
Should we not use QUIC_CONNECTION_HANDSHAKE_TIMEOUT from solana-quic-definitions instead?

Reply:
That  value in solana-quic-definitions is set to the same as the old connection idle timeout 60 (s). We need shorter connection handshake timeout here.

Reply:
that would make timeouts in connection cache and tpu-client-next inconsistent. is it not better to keep them in sync?